src/bundle/Resources/config/routing.yaml

```diff
@@ -555,13 +555,27 @@ ezplatform.content.translate:
         _controller: 'Ibexa\Bundle\AdminUi\Controller\ContentEditController::translateAction'
         fromLanguageCode: ~
 
+ibexa.content.translate_with_location:
+    path: /content/{contentId}/location/{locationId}/translate/{toLanguageCode}/{fromLanguageCode}
+    methods: ['GET', 'POST']
+    defaults:
+        _controller: 'EzSystems\EzPlatformAdminUiBundle\Controller\ContentEditController::translateAction'
```

Comment:
```suggestion
        _controller: 'Ibexa\Bundle\AdminUi\Controller\ContentEditController::translateAction'
```

Reply:
As discussed, due to https://github.com/ezsystems/ezplatform-admin-ui/blob/5c35adbdf2805fe16078be9f3fc7583281fe8f23/src/lib/View/Builder/ContentTranslateViewBuilder.php#L72 it has to be left as is.